frontend/src/services/http.js

```diff
@@ -3,10 +3,10 @@ import { AuthService } from './auth';
 
 const trim = (string) => {
   if (string.startsWith('/')) {
-    return string.slice(1);
```

Comment:
Good catch, end up returning without trimming the trailing slash when this returns.